src/data/init/init-moves.ts

```diff
@@ -3476,7 +3476,7 @@ export function initMoves() {
       .attr(TeraMoveCategoryAttr)
       .attr(TeraStarstormTypeAttr)
       .attr(VariableTargetAttr, (user, _target, _move) =>
-        user.species.speciesId === SpeciesId.TERAPAGOS && user.isTerastallized()
+        user.species.speciesId === SpeciesId.TERAPAGOS && user.isTerastallized
```

Comment:
NIT: If you dont' need params after the optionals you can omit them:
```suggestion
     .attr(VariableTargetAttr, (user) =>
        user.species.speciesId === SpeciesId.TERAPAGOS && user.isTerastallized
```